staging/src/k8s.io/apimachinery/pkg/util/net/util.go

```diff
@@ -54,3 +54,20 @@ func IsConnectionReset(err error) bool {
 	}
 	return false
 }
+
+// Returns if the given err is "connection refused" error
+func IsConnectionRefused(err error) bool {
```

Comment:
Here is another way of doing it: https://github.com/openshift/openshift-azure/blob/master/pkg/util/errors/errors.go
(not saying you have to do it that way... just fyi)

Reply:
copy that, thx